src/entities/ProposalSurveyTopics/utils.ts

```diff
@@ -0,0 +1,15 @@
+import RequestError from 'decentraland-gatsby/dist/entities/Route/error'
+import { Request } from 'express'
+
+import ProposalModel from '../Proposal/model'
+import SurveyTopicModel from '../SurveyTopic/model'
+
+export async function getProposalSurveyTopics(req: Request<{ proposal: string }>) {
+  try {
+    const id = req.params.proposal
+    const proposal = await ProposalModel.getProposal(id)
```

Comment:
If we pass the category instead of the proposal we can avoid the getProposal call here.